app/scripts/modules/core/application/service/applications.read.service.js

```diff
@@ -92,14 +95,10 @@ module.exports = angular
           return taskLoader
             .then(function(tasks) {
               addTasksToApplication(application, tasks);
+              application.taskRefreshStream.onNext();
               application.tasksLoaded = true;
               application.tasksLoading = false;
               application.tasksLoadFailure = false;
-              if (application.tasksLoaded || forceReload) {
-                $rootScope.$broadcast('tasks-reloaded', application);
-              } else {
-                $rootScope.$broadcast('tasks-loaded', application);
-              }
```

Comment:
The `-loaded`/`-reloaded` thing was either refactored out of relevance or aspirational; either way, we are not listening for either message in the codebase now.